cmd/vm_show.go

```diff
@@ -99,6 +101,14 @@ func showVM(name string) (outputter, error) {
 		out.AntiAffinityGroups[i] = aag.Name
 	}
 
+	for _, nic := range vm.Nic {
+		if nic.ID == vm.DefaultNic().ID {
+			continue
+		}
```

Comment:
It seems to be a convoluted way of doing.
```golang
if nic.IsDefault {
    continue
}
```

https://github.com/exoscale/egoscale/blob/master/virtual_machines.go#L146

Reply:
Thanks good catch! I did not see that 😉